invenio_vocabularies/contrib/affiliations/config.py

```diff
@@ -26,7 +26,16 @@ class AffiliationsSearchOptions(SearchOptions):
     """Search options."""
 
     suggest_parser_cls = SuggestQueryParser.factory(
-        fields=["name^100", "acronym^50", "title.*^20", "id^20", "aliases^20"],
+        fields=[
+            "name.suggest^100",
+            "name^80",
+            "acronym.suggest^50",
+            "acronym^40",
+            "title.*^20",
+            "id^20",
+            "aliases^20",
+        ],
+        fuzziness="AUTO",
```

Comment:
could you add a small reference to docs here so we can check what fuzziness auto does? (in the code comment)